parachain-template/node/Cargo.toml

```diff
@@ -26,75 +26,75 @@ serde = { version = "1.0.119", features = [ "derive" ] }
 flume = "0.10.9"
 
 # RPC related Dependencies
-jsonrpsee = { version = "0.14.0", features = ["server", "macros"] }
+jsonrpsee = { version = "0.15.0", features = ["server", "macros"] }
```

Comment:
this will alkso be bumped in the manta PR i suppose?

Reply:
Yeah it has to be on manta's side or it won't compile